ygot/pathtranslate/pathtranslate.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package pathtranslate exports api to transform given string slice into
+// different forms in a schema aware manner.
+package pathtranslate
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/openconfig/goyang/pkg/yang"
+
+	gnmipb "github.com/openconfig/gnmi/proto/gnmi"
+)
+
+var (
+	seperator = "/"
```

Comment:
Sorry, just spotted this nit.

Should we make this a `const`, and also `s/seperator/separator/g` in this file.

Reply:
Thanks for catching this. Addressed in the next commits.